src/components/Select/Select.tsx

```diff
@@ -0,0 +1,364 @@
+/** @jsxRuntime classic /*
+/** @jsx jsx */
+import React, { useContext, useMemo } from "react"
+import { HerzUITheme, jsx } from "theme-ui"
+
+import { useSelect, SELECT_BULK_ACTIONS } from "./hooks/useSelect"
+import { Button, Popover, Icon } from ".."
+import { InputGroupContext } from "../InputGroup/Context"
+import { getDataFromChildren, isArrayEqual } from "./utils"
+import { SelectOption } from "./SelectOption"
+import { SelectContext } from "./context"
+
+export type SelectValue = string | number | Record<string, unknown>
+export type SelectedItems = Array<SelectValue>
+
+export type SelectOption = {
```

Comment:
You have two variables with the same name SelectOption.

Reply:
Fixed in 1029c24